web-common/src/features/themes/color-utils.ts

```diff
@@ -0,0 +1,78 @@
+import type { V1Color } from "@rilldata/web-common/runtime-client";
+
+export type ThemeColor = [number, number, number];
```

Comment:
It seems like `ThemeColor` is being used to both represent RGB colors and HSL colors. It’d be clearer if there were different types for `RBGColor` and `HSLColor`.